src/style-spec/reference/v8.json

```diff
@@ -861,6 +861,9 @@
         },
         "line": {
           "doc": "The label is placed along the line of the geometry. Can only be used on `LineString` and `Polygon` geometries."
+        },
+        "line-center": {
+          "doc": "The label is placed at the center of the line of the geometry. Can only be used on `LineString` and `Polygon` geometries. Note that a single feature in a vector tile may contain multiple line geometries. The center is calculated based on the entirety of the geometry included in the tile buffers, and the label may extend past the edge of the tile."
```

Comment:
@samanpwbb you may want to review this description, since Studio will display this text for the property value tooltip.

Reply:
This description is a little on the long side. Is the last sentence necessary?